flink-table/flink-table-planner/src/main/java/org/apache/flink/table/operations/TableOperationToRelNodeConverter.java

```diff
@@ -120,4 +144,28 @@ public RelNode visitOther(TableOperation other) {
 				.collect(toList());
 		}
 	}
+
+	private class AggregateVisitor extends ExpressionDefaultVisitor<AggCall> {
+
+		@Override
+		public AggCall visitCall(CallExpression call) {
+			if (call.getFunctionDefinition() == AS) {
+				String aggregateName = extractValue(call.getChildren().get(1),
```

Comment:
Should we move `extractValue` and `isFunctionOfType` to `ExpressionUtils` in `common`? Because it has nothing to do with API similar to `ExpressionDefaultVisitor`?